idunn/utils/categories.yml

```diff
@@ -1,144 +1,602 @@
+          - "fast_food,*"
+        pj_what: "fast food"
+
     bar:
-        pj_what: "bars"
+        regex: "^(cafes?|bars?|pubs?)( |$)"
+        raw_filters:
+          - "bar,*"
+          - "cafe,*"
+          - "pub,*"
+          - "biergarten,*"
+        pj_what: "bar"
         pj_filters:
-            - "cafés, bars"
+          - "cafés, bars"
+
+    hotel:
+        regex: "hotel"
         raw_filters:
-            - "bar,*"
-            - "cafe,*"
-        regex: "^(cafes?|bars?|pubs?)( |$)"
-    museum:
-        pj_what: "musée"
+          - "*,hotel"
+          - "*,chalet"
+          - "*,hostel"
+          - "*,alpine_hut"
+          - "*,motel"
+          - "*,bed_and_breakfast"
+          - "*,guest_house"
+          - "*,dormitory"
+        pj_what: "hôtels"
         pj_filters:
-            - "musées"
+          - "hôtels"
+
+    historic:
         raw_filters:
-            - "museum,*"
-        regex: "^(museum|musee)"
-    health:
-        pj_what: "hopital"
+          - "memorial,*"
+          - "monument,*"
+          - "statue,*"
+          - "castle,*"
+          - "citywalls,*"
+          - "fort,*"
+          - "battlefield,*"
+          - "castle_walls,*"
+          - "earthworks,*"
+          - "moat,*"
+          - "church,*"
+          - "tumulus,*"
+          - "stone_circle,*"
+          - "menhir,*"
+          - "standing_stone,*"
+          - "bridge,*"
+          - "milestone,*"
+          - "wall,*"
+          - "well,*"
+          - "folly,*"
+          - "ruins,*"
+        pj_what: "monument historique"
+
+    post_office:
+        raw_filters:
+          - "post_office,*"
+        pj_what: "bureau de poste"
+
+    fuel:
+        raw_filters:
+          - "fuel,*"
+        pj_what: "station essence"
+
+    community_centre:
+        raw_filters:
+          - "*,community_centre"
+        pj_what: "centre communal"
+
+    shop_convenience:
+        raw_filters:
+          - "*,convenience"
+          - "*,deli"
+        pj_what: "épicerie"
+
+    shop_car:
+        raw_filters:
+          - "*,car_repair"
+          - "*,car"
+        pj_what: "concessionnaire"
+
+    kindergarten:
+        raw_filters:
+          - "*,kindergarten"
+        pj_what: "crèche"
+
+    camp_site:
+        raw_filters:
+          - "camp_site,*"
+          - "caravan_site,*"
+        pj_what: "camping"
+
+    station:
+        regex: "^gare( |$)"
+        raw_filters:
+          - "railway,station"
+          - "railway,tram_stop"
+          - "railway,halt"
+        pj_what: "gare"
+
+    shop_butcher:
+        raw_filters:
+          - "*,butcher"
+        pj_what: "boucher"
+
+    attraction:
+        raw_filters:
+          - "nightclub,*"
+          - "leisure,escape_game"
+          - "leisure,golf_course"
+          - "leisure,miniature_golf"
+          - "tourism,aquarium"
+          - "tourism,artwork"
+          - "tourism,attraction"
+          - "tourism,theme_park"
+          - "tourism,zoo"
+        pj_what: "loisir"
```

Comment:
Same question about "pj_what". Does that provide relevant results ? And then, could it be used for "leisure" too ?